common/common.scss

```diff
@@ -212,3 +212,37 @@ input {
 .alert-emails-disabled {
     display: none;
 }
+
+.badge-card {
+  background-color: #FFFFFF;
+  border: 1px solid #E9EAEE;
+  
+  .medium:hover {
+    box-shadow: 0px 6px 12px rgba(0, 0, 0, 0.083);
+  }
+
+  .grant-count {
+    right: unset;
+    left: 10%;
+    top: 16px;
+    color: #E9EAEE;
+    font-family: 'Helvetica Neue', Arial, sans-serif;
+    font-weight: normal;
+    font-size: 4.5em;
+  }
+
+  .badge-contents {
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 24px 10%;
+    .badge-icon {
+      width: 100%;
+      justify-content: flex-end;
+      margin-bottom: -1rem;
+    }
+    .badge-info {
+      padding: 1em 0 0 0;
+      flex: unset;
```

Comment:
discourse sets its own `flex`, this is to undo its effects.

Reply:
Maybe better to reset it to `initial` if we're pretty sure of removing the flex effect since `unset` may introduce an inexplicit result according to the inherited behaviour from its parent elements.

https://www.digitalocean.com/community/tutorials/css-inherit-initial-unset#:~:text=initial%3A%20The%20default%20value%20for,it%20will%20act%20as%20initial.